webcrypto/aes.go

```diff
@@ -544,7 +544,7 @@ func (agp *AESGCMParams) Decrypt(ciphertext []byte, key CryptoKey) ([]byte, erro
 	// 2.
 	// Note that we multiply the length of the ciphertext by 8, in order
 	// to get the length in bits.
-	if len(ciphertext)*8 < tagLength {
+	if int(byteLength(len(ciphertext)).asBitLength()) < tagLength {
```

Comment:
why not make `tagLength` a `bitLength` instead?

Reply:
Good call, I completely overlooked it :+1: